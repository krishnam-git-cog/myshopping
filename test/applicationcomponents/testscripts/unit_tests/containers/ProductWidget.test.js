import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import ProductWidget from '../../../../../src/containers/productswidget/ProductWidget';
import * as reportfactory from '../../../../utilities/reportfactory'

describe('ProductWidget test', () => {
    it("Container - ProductWidget should match snapshot", () => {
        const data = [
            {
                "id": 1234561,
                "productCode": "a214a774ce2a4e52b8fd85608458b59c",
                "productName": "Apple iPhone 6s Plus",
                "productImage": "Apple-iPhone-6s-Plus.jpg",
                "productDescription": "Lorem ipsum dolor sit amet",
                "productPrice": 488.8,
                "productQuantity": 20,
                "inStock": true
            },
            {
                "id": 1234562,
                "productCode": "83e50bc3a2ce47308ae8959720c3a206",
                "productName": "Apple iPhone 7",
                "productImage": "Apple-iPhone-7.jpg",
                "productDescription": "Lorem ipsum dolor sit amet",
                "productPrice": 498,
                "productQuantity": 20,
                "inStock": true
            }
        ];

        let shallow = createShallow();
        const component = shallow(<ProductWidget data={data}/>);
        expect(component).toMatchSnapshot();
    });

});