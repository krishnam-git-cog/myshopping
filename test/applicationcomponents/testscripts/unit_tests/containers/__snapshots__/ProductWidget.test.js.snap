// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductWidget test Container - ProductWidget should match snapshot 1`] = `ShallowWrapper {}`;
